libgalois/include/katana/PropertyGraph.h

```diff
@@ -136,6 +131,23 @@ class KATANA_EXPORT GraphTopology {
 
   bool empty() const noexcept { return num_nodes() == 0; }
 
+  ///@param node node to get degree for
+  ///@returns Degree of node N
+  size_t degree(Node node) const noexcept { return edges(node).size(); }
```

Comment:
I don't think so, because, with inlining the compiler should be able to reduce iterator subtraction to subtraction between 2 numbers (counting iterator is after all, a thin wrapper).